tests/unit/test-fdlibm.c

```diff
@@ -0,0 +1,83 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
+ * Copyright 2016 University of Szeged.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Unit test for fdlibm
+ */
+
+#include "test-common.h"
+
+static bool passed = true;
+
+static void
+check_int (const char *expr, int computed, int expected)
+{
+  printf ("%s = %d [expected=%d] ", expr, computed, expected);
+
+  bool result = computed == expected;
+  printf ("%s\n", result ? "PASS" : "FAIL");
```

Comment:
I would only print result information if:
a) I'm running the unit tests with verbose option to see all results (I know we don't have such option now)
b) or there is a failure.